R/themes.R

```diff
@@ -36,6 +36,9 @@
 #' # Helpful if you want to know the Bootstrap version of the current theme
 #' theme_version()
 #'
+#' # Can create a theme and return the object, instead of setting it globally.
+#' bs_theme_create()
+#'
```

Comment:
If we're going to include `bs_theme_create()` in the examples, let's show how one might use it...In fact, let's move this down and replace lines 56-66 with something like:

```r
bs3 <- bs_theme_create(3)
red_default <- sass::sass_layer(defaults = list("brand-primary" = "red !default"))
bs3 <- sass::sass_layer_merge(bs3, red_default)
bootstrap_sass(".my-class { color: $brand-primary; }", bs3)
```

Reply:
Would it be better to have the `bs_theme_add` function take a `.theme` argument, and then make an example with that? (That would be a separate PR)

Reply:
Maybe, but I still think that it's useful to point out that themes are just sass layers.

BTW, I looked a little into adding a `.theme` argument yesterday and it's not as trivial as I was hoping